### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader: NVDA does not announcing the table name it is only announcing as table.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1889]
**Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**

**GitHubTags**: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;#NVDA; #Win10;

**Environment Details:**
Desktop App: Windows Version: Windows10
NVDA Version: 2019.1.1

**Steps to Reproduce:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Table under the tool bar.
7. Verify that all the elements of "DiskInfoTable" Screen are accessible and MAS Compliant.

 **Actual** 
NVDA does not announcing the table name it is only announcing as table.

**Same issue is observed with JAWS for below scenarios.
Blob Container Table : Table-**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and Upload a blob/file by using  "Upload Files" control.
6. Navigate to Table present in right side of the storage explorer.
7. Verify that all the elements of "Table" screen are accessible and MAS Compliant.

**Expected** 
NVDA should announce  the table name as "Disk info table".

**Recommendations:**
**Refer below link which is repository of bug fixes code snippets:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx  

**User Impact:**
Screen reader user unable to know that about which table it is reading.

**MAS Reference:
MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 
Attachment for Reference
![1889_A11y_AzureToolsStorageExplorer_Disk_DiskInfoTable_ScreenReader](https://user-images.githubusercontent.com/55691739/65484347-b39adf00-debc-11e9-99a6-a96cc6011584.png)

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
